docker/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # It is not recommended for use beyond testing scenarios.
 ##############################################################################
 
-FROM fedora:31
+FROM fedora:33
 MAINTAINER Luke Hinds <lhinds@redhat.com>
```

Comment:
Would it be an idea to use an email list for Keylime here perhaps?

Reply:
Yeah, that's probably a good idea to do in a lot of places. We have a domain name, so we should be able to get some emails for it and forward those to different people at different times.